### PR TITLE
Update ui build to use 'admin' folder instead of 'core'

### PR DESCRIPTION
**Pull request by gsusmi**
Post release of v0.1.2, ui/core folder has been renamed to ui/admin.
Update ui build scripts to be aware of the path change.

@samsalisbury , I wasn't able to figure out how to update folder path reference in `packages-oss.lock`. Is there a command I should be running to get it updated? Appreciate your help in figuring this out.

packages-oss.lock/pkgs.yml
```
724:          && UIDIR="$DIR/ui/core/dist" \
```
should read:
```
724:          && UIDIR="$DIR/ui/admin/dist" \
```

### CODE REVIEW

File: internal/ui/VERSION [hunk 1]
```diff
@@ -1,4 +1,4 @@
-04770f4e50b84b3ec712ba2b5c09a0c03fdbea8c Merge pull request #372 from hashicorp/chore-alternate-dropdown-radios
+b7937e77ad5a37fcf0bf0ed2b6d4e4b34d97b144 Merge pull request #383 from hashicorp/rename-core-to-admin
```
Comment (gsusmi) [inline internal/ui/VERSION:1]: Bumped up ui commit to latest to pick up folder change in boundary-ui repo.

**Comment by samsalisbury** [comment]
Hi @gsusmi yes, after modifying the spec file `packages-oss.yml` you need to run `make packages` to regenerate the build system and CI config. I will send you links to documentation on Slack.

**Review by samsalisbury — CHANGES_REQUESTED** [review]
Needs `make packages` running as discussed, thanks!

**Comment by gsusmi** [comment]
Update includes .circleci and package-oss.lock changes after running `make packages`.

**Review by samsalisbury — APPROVED** [review]
:shipit:

**Pull request merged**